launch/model_endpoint.py

```diff
@@ -236,9 +242,10 @@ def predict(self, request: EndpointRequest) -> EndpointResponse:
         )
         return EndpointResponse(
             client=self.client,
-            status=TASK_SUCCESS_STATE,
+            status=raw_response.get("state"),
```

Comment:
should we consolidate on state vs status haha

Reply:
I think we should tbh, I like status personally